utils/components.py

```diff
@@ -59,12 +59,23 @@ def list_models_in_directory_op(models_folder: str) -> List[str]:
     return models
 
 
-@dsl.component(
-    base_image=PYTHON_IMAGE,
-    install_kfp_package=False,
-    packages_to_install=["huggingface_hub"],
-)
-def huggingface_importer_op(repo_name: str, model_path: str = "/model"):
-    from huggingface_hub import snapshot_download
+@dsl.container_component
+def model_to_pvc_op(model: dsl.Input[dsl.Model], pvc_path: str = "/model"):
+    return dsl.ContainerSpec(
+        TOOLBOX_IMAGE,
+        ["/bin/sh", "-c"],
+        [f"cp -r {model.path}/* {pvc_path}"],
+    )
+
 
-    snapshot_download(repo_id=repo_name, cache_dir="/tmp", local_dir=model_path)
+@dsl.container_component
+def ilab_importer_op(repository: str, release: str, base_model: dsl.Output[dsl.Model]):
+    return dsl.ContainerSpec(
+        RHELAI_IMAGE,
+        ["/bin/sh", "-c"],
+        [
+            "echo 'Downloading model...' && "
+            + "echo '{}' > $HOME/config.yaml && "
```

Comment:
This shouldn't be needed.

Reply:
It actually is. `ilab model download` requires `ilab config init` but at the same time, we don't need the config. So we need to fake it.

Reply:
I don't understand why it is needed, I can download a model by using `--config=DEFAULT`  without having a config file on my system.
What's the error you're seeing?

Reply:
Oh.. I didn't realize `--config=DEFAULT` means something else than without `--config` specified. I didn't even know such magic value exists. It's not documented in InstructLab as far as I know... I'll try that, thanks.